docs/build/eps.md

```diff
@@ -634,8 +634,8 @@ See more information on the MIGraphX Execution Provider [here](../execution-prov
 ### Prerequisites
 {: .no_toc }
 
-* Install [ROCM](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html)
-  * The MIGraphX execution provider for ONNX Runtime is built and tested with ROCM3.3
+* Install [ROCm](https://docs.amd.com/bundle/ROCm-Installation-Guide-v5.4/page/How_to_Install_ROCm.html#_How_to_Install)
+  * The MIGraphX execution provider for ONNX Runtime is built and tested with ROCM5.4
```

Comment:
>MIGraphX [](http://example.com/codeflow?start=8&length=8)

ROCm?

Reply:
This is under the MIGraphx Section I believe.